users/ignoring.rst

```diff
@@ -34,62 +34,77 @@ Patterns
-   included more than once. Note that while this can be used to include
-   patterns from a file in a subdirectory, the patterns themselves are
-   still relative to the folder *root*. Example:
-   ``#include more-patterns.txt``.
-
--  A pattern beginning with a ``!`` prefix negates the pattern: matching files
-   are *included* (that is, *not* ignored). This can be used to override
-   more general patterns that follow.
-
--  A pattern beginning with a ``(?i)`` prefix enables case-insensitive pattern
-   matching. ``(?i)test`` matches ``test``, ``TEST`` and ``tEsT``. The
-   ``(?i)`` prefix can be combined with other patterns, for example the
-   pattern ``(?i)!picture*.png`` indicates that ``Picture1.PNG`` should
-   be synchronized. On Mac OS and Windows, patterns are always case-insensitive.
-
--  A pattern beginning with a ``(?d)`` prefix enables removal of these files if
-   they are preventing directory deletion. This prefix should be used by any OS
-   generated files which you are happy to be removed.
-
--  A line beginning with ``//`` is a comment and has no effect.
-
--  Windows does not support escaping ``\[foo - bar\]``.
-
-.. note::
-
-   Prefixes can be specified in any order (e.g. "(?d)(?i)"), but cannot be in a
+Regular file names match themselves, i.e. the pattern foo matches the files ``foo``,
+``subdir/foo`` as well as any directory named ``foo``. Spaces are treated as regular characters.
+
++-------------+-----------+----------+-------------------------------------------------------------------------------+
+| Character   | Where     | Affects  | Action                                                                        |
++=============+===========+==========+===============================================================================+
+| ``/``       | Beginning | Folders  | A pattern beginning with ``/`` matches in the current directory only.         |
+|             |           |          | ``/foo`` matches ``foo`` but not ``subdir/foo``.                              |
+|             |           |          |                                                                               |
++-------------+-----------+----------+-------------------------------------------------------------------------------+
+| ``!``       | Beginning | All      | A pattern beginning with a ``!`` prefix negates the pattern: matching files   |
+|             |           |          | are *included* (that is, *not* ignored). This can be used to override         |
+|             |           |          | more general patterns that follow.                                            | 
+|             |           |          | Include patterns (that begin with ``!``) cause Syncthing to traverse and      |
+|             |           |          | :ref:`watch <scanning>` the entire directory tree regardless of other         |
+|             |           |          | ignore patterns.                                                              |
```

Comment:
That could be misunderstood or at least confuse as to whether the other patterns apply still. I'd just put a reference to the note below here. And then in the note explicitly state that the ignore patterns still apply to the results of scanning/watching.